electron/legendary/eos_overlay/eos_overlay.ts

```diff
@@ -198,7 +203,7 @@ async function isEnabled(appName?: string, runner?: Runner) {
   let enabled = false
 
   let prefix = ''
-  if (isLinux || !(appName && runner)) {
+  if (!isWindows && isLinux && appName && runner) {
```

Comment:
The previous check was not working on Windows, it was checking for a winePrefix so it was receiving an empty object and passing to legendary.

Now if it is not Windows it also needs to have an appName and Runner.
Let me know if this makes sense or not @CommandMC

Reply:
Right, I see the issue, not sure how that `!` got there.

The `isWindows` check is redundant, `isLinux` and `isWindows` will never both be true at the same time.
```suggestion
  if (isLinux && appName && runner) {
```

Reply:
That's totally true 👍🏽